### PR TITLE
Charging system improvements

### PR DESCRIPTION
**Pull request by gestom**
1) Docking station position parametrized - allows to use maps with arbitrary docking station position.
2) Circle detection sensitivity increased - allows to dock in adverse lighting conditions.
3) Initial stages of docking do not check if the robot behaves as expected anymore (this was included to verify if other nodes still send commands to the robot or if the robot was moving when charging was initiated) - improves interaction with higher navigation layers that seemed to activate the charging while the robot was still moving.

### CODE REVIEW

**Comment by marc-hanheide** [comment]
what is "rosie_talker" and why is it included?

**Comment by marc-hanheide** [comment]
also `scitos_ptu/src/scripts/ptu_action_server_metric_map.py` seems to be unrelated?

File: scitos_docking/launch/charging.launch [hunk 1]
```diff
@@ -1,13 +1,17 @@
 <launch>
   <arg name="machine" default="localhost" />
   <arg name="user" default="" />
-  <arg name="lightEBC" default="Port0_12V_Enabled" />
```
Comment (marc-hanheide) [inline scitos_docking/launch/charging.launch]: Wasn't this a problem for some people who experienced their side PCs to be shut down? Sorry, I'm not quite sure.

File: scitos_docking/launch/charging_mux.launch [hunk 2]
```diff
@@ -1,12 +1,15 @@
 <launch>
   <arg name="machine" default="localhost" />
   <arg name="user" default="" />
-  <arg name="lightEBC" default="Port0_12V_Enabled" />
```
Comment (marc-hanheide) [inline scitos_docking/launch/charging_mux.launch]: see above? Why this change?

**Comment by marc-hanheide** [comment]
Now I understand the problem... @gestom you need to first update (i.e. `git pull https://github.com/strands-project/scitos_apps.git hydro-devel`) your fork. Right now this PR is based on very much outdated code...

**Comment by gestom** [comment]
This (docking) code is the one that is used on Linda. I took it directly from her.

**Comment by marc-hanheide** [comment]
That might well be, but your fork is based on version 0.0.9 of scitos_apps, while we are now at 0.0.14. We cannot merge this. You should update your fork (happy to help).

**Comment by marc-hanheide** [comment]
All clean now

**Comment by gestom** [comment]
thanks for the help
